### PR TITLE
RavenDB-17649 Restart database on a single node button

### PR DESCRIPTION
**Pull request by kalczur**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17649/Restart-database-on-a-single-node-button

### Type of change

- New feature

### How risky is the change?

- Low 

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work
image
image

### CODE REVIEW

**Comment by kalczur** [comment]
I'll wait until [RavenDB-20367](https://issues.hibernatingrhinos.com/issue/RavenDB-20367) is merged to avoid conflicts.

**Comment by karmeli87** [comment]
more conflicts... probably I merged too many PRs Y-day :)

File: src/Raven.Studio/typescript/commands/resources/restartDatabaseCommand.ts [hunk 1]
```diff
@@ -1,20 +1,23 @@
 import commandBase = require("commands/commandBase");
-import database = require("models/resources/database");
 import endpoints = require("endpoints");
+import { DatabaseSharedInfo } from "components/models/databases";
 
 class restartDatabaseCommand extends commandBase {
+    private db: DatabaseSharedInfo;
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/resources/restartDatabaseCommand.ts:6]: readonly

File: src/Raven.Studio/typescript/commands/resources/restartDatabaseCommand.ts [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
+    private location: databaseLocationSpecifier;
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/resources/restartDatabaseCommand.ts:7]: readonly

File: src/Raven.Studio/typescript/components/pages/database/indexes/list/useIndexesPage.tsx [hunk 3]
```diff
@@ -274,39 +280,41 @@ export function useIndexesPage(database: database, stale: boolean) {
...
-                    if (indexStatus === "Disabled") {
-                        await indexesService.enable(index, database, location);
-                    } else {
-                        await indexesService.resume(index, database, location);
+            for (const index of indexes) {
+                for (const { nodeTag, shardNumbers } of contexts) {
+                    const locations = ActionContextUtils.getLocations(nodeTag, shardNumbers);
+
+                    for (const location of locations) {
+                        const indexStatus = index.nodesInfo.find((x) => _.isEqual(x.location, location))?.details
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/indexes/list/useIndexesPage.tsx:299]: IIRC there is databaseLocationComparator

**Review by ml054 — APPROVED** [review]

**Pull request merged**